test/mobile/test_upgrader_codegen.py

```diff
@@ -11,15 +11,14 @@
 import os
 from torch.jit.generate_bytecode import generate_bytecode
 
-pytorch_test_dir = Path(__file__).resolve().parents[1]
+pytorch_caffe2_dir = Path(__file__).resolve().parents[2]
```

Comment:
would this still work with OSS?

Reply:
Yeah in OSS, it will point to the root folder like `/Users/chenlai/pytorch`.